packages/legend-graph/src/graph/DependencyManager.ts

```diff
@@ -131,6 +136,18 @@ export class DependencyManager {
     return this.dependencyGraphs.flatMap((dep) => dep.allOwnElements);
   }
 
+  get origin(): GraphDataOrigin | undefined {
+    return this._origin;
+  }
+
+  setOrigin(val: GraphDataOrigin): void {
+    if (this._origin) {
+      throw new IllegalStateError(`Graph Origin has already been set`);
```

Comment:
Let's normalize the casing. Also, can you update the message we had in `BasicModel`? 

```suggestion
      throw new IllegalStateError(`Graph origin has already been set`);
```